x-pack/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -38,14 +38,14 @@ import { getAuthorizationHeader } from '../lib/request_authorization';
 import type { MlClient } from '../lib/ml_client';
 import type { DataViewsService } from '../../../../../src/plugins/data_views/common';
 
-function getIndexPatternId(dataViewsService: DataViewsService, patternName: string) {
-  const iph = new IndexPatternHandler(dataViewsService);
-  return iph.getIndexPatternId(patternName);
+function getDataVIewId(dataViewsService: DataViewsService, patternName: string) {
```

Comment:
Looks like an uppercase `I` in `VIew` by mistake here.

Reply:
Updated in https://github.com/elastic/kibana/pull/116455/commits/7ee9241872df9cecf890ae7dce916bfa3f491d84